commands/pause.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+    name: "pause",
+    aliases: ["pause", "hold"],
+    run: async (client, message, args) => {
+        if (!message.member.voice.channel) return message.channel.send(`${client.emotes.error} | You must be in a voice channel!`)
+        let queuethere = client.distube.getQueue(message);
+         if (!queuethere) return message.channel.send(`${client.emotes.error} There is nothing in the queue right now!`)
+         client.distube.pause(message)
```

Comment:
I think we should unpause the song if it have been paused already.
```suggestion
        let queue = client.distube.getQueue(message)
        if (!queue) return message.channel.send(`${client.emotes.error} | There is nothing in the queue right now!`)
        if (queue.pause) {
            client.distube.resume(message)
            return message.channel.send("Resumed the song for you :)")
        }
        client.distube.pause(message)
```